_posts/2024-04-06-why-no-release.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Why no release (or nightly) yet?
+author: Rubidium
+---
+
+On the 29th of March a post on a security mailing list notified the world about a backdoor in a library.
```

Comment:
"a library" ? Any reason why we can't be a bit more explicit?

Reply:
"xz" is mentioned nowhere in the post...

Reply:
There's no strong reason why it can't be more explicit.